di-ipv-cimit-stub/lib/src/main/java/uk/gov/di/ipv/core/library/config/EnvironmentVariable.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.di.ipv.core.library.config;
+
+public enum EnvironmentVariable {
```

Comment:
As per the comments on SAM template, I'd have a single `CIMIT_CONFIG_BASE_PATH` environment variable whose value is read and prepended to the `signingKey` and `componentId` parameter properties in `ConfigService`